JavaScriptSPA/index.html

```diff
@@ -2,58 +2,61 @@
 <html>
 <head>
     <title>Quickstart for MSAL JS</title>
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/bluebird/3.3.4/bluebird.min.js"></script>
     <script src="https://secure.aadcdn.microsoftonline-p.com/lib/0.2.3/js/msal.js"></script>
-    <script src="https://code.jquery.com/jquery-3.2.1.min.js" class="pre"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/bluebird/3.3.4/bluebird.min.js" class="pre"></script>
-    <style>
-        table, td, th {
-            border: 1px solid black;
-        }
-    </style>
+    <script src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
 </head>
 
 <body>
     <h2>Welcome to MSAL.js Quickstart</h2><br/>
     <h4 id="WelcomeMessage"></h4>
     <button id="SignIn" onclick="signIn()">Sign In</button><br/><br/>
-    <table id="graphData"></table>
+	<pre id="json"></pre>
 
     <script>
+		// Browser check variables
+		var ua = window.navigator.userAgent;
+		var msie = ua.indexOf('MSIE ');
+		var msie11 = ua.indexOf('Trident/');
+		var msedge = ua.indexOf('Edge/');
+		var isIE = msie > 0 || msie11 > 0;
+		var isEdge = msedge > 0;
+	
         var applicationConfig = {
             clientID: "Enter_the_Application_Id_here", //This is your client ID
             graphScopes: ["user.read"],
             graphEndpoint: "https://graph.microsoft.com/v1.0/me"
         };
 
         //Pass null for default authority (https://login.microsoftonline.com/common) and tokenReceivedCallback if using popup apis
-        var myMSALObj = new Msal.UserAgentApplication(applicationConfig.clientID, null, null);
-
+        var myMSALObj = new Msal.UserAgentApplication(applicationConfig.clientID, null, null, {storeAuthStateInCookie:true, cacheLocation:"localStorage"});
```

Comment:
I don't think we want to set the storeAuthStateInCookie and cacheLocation by default. This should only be necessary for those who use Edge and is documented in the wiki.

Reply:
Per @andretms , we need to set redirectURI to location.origin for it to work in VS project.